src/css/stories/InteractionDesigner.css

```diff
@@ -0,0 +1,496 @@
+/*
+* Helper functions for layout
+* Useful for storybook style
```

Comment:
I don't know why we lost this file in current master, we've been having it since the beginning though